addons/base_import/models/base_import.py

```diff
@@ -566,7 +566,11 @@ def _extract_header_types(self, preview_values, options):
                 val = self._remove_currency_symbol(val)
                 if val:
                     if options.get('float_thousand_separator') and options.get('float_decimal_separator'):
-                        val = val.replace(options['float_thousand_separator'], '').replace(options['float_decimal_separator'], '.')
+                        if options.get('float_decimal_separator') == '.' and val.count('.') > 1:
+                            # This is not a float so exit this try
+                            float('a')
+                        else:
+                            val = val.replace(options['float_thousand_separator'], '').replace(options['float_decimal_separator'], '.')
```

Comment:
this could be outside of the else (and minimize the diff) but both are equivalent